.github/workflows/python-package.yml

```diff
@@ -31,10 +31,10 @@ jobs:
         pip install pytest
         pip install coverage
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
-    - name: Format code with black
-      run: |
-        # stop the build if code does not conform to black formatting
-        black . --check --line-length 79 --exclude "/versioneer.py|pdbx/_version.py"
+    # - name: Format code with black
+    #   run: |
+    #     # stop the build if code does not conform to black formatting
+    #     black . --check --line-length 78 --exclude "/versioneer.py|pdbx/_version.py"
```

Comment:
I would recommend creating a pyproject.toml with the content below at the root directory. This will allow you just run `black . --check` and remove all the other parameters that cause problems (especially versioneer).

```
[tool.black]
line-length = 79
target-version = ['py37']
include = '\.pyi?$'
exclude = '''
(
  /(
      \.eggs         # exclude a few common directories in the
    | \.git          # root of the project
    | \.mypy_cache
    | \.tox
    | \.venv
    | _build
    | buck-out
    | build
    | dist
  )/
  | versioneer.py
  | mmcif_pdbx/_version.py  # Exclude files from versioneer
)
'''
```

Reply:
I did but I could not get it to work.  It was flagging problems my local installation of `black` wasn't catching.